JSDemos/Demos/Charts/Annotation/description.md

```diff
@@ -1,6 +1,6 @@
 Annotations are images and text blocks that provide additional information on the visualized data.
 
-To configure annotations, assign an array of objects to the [annotations](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/annotations/) option. Each object configures an individual annotation. To specify options for all annotations, use the [commonAnnotationSettings](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/commonAnnotationSettings/) object. Individual settings take precedence over these settings.
+To configure individual annotations, assign an array of objects to the [annotations](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/annotations/) option. Each object configures an individual annotation. To specify options for all annotations, use the [commonAnnotationSettings](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/commonAnnotationSettings/) object. Individual settings take precedence over these settings.
```

Comment:
I made a change to a similar block in another merge request. I don't know why you add the word "individual" if you then switch to an "array". Then the next sentence talks about "individual" annotations again (repetition). 

Here's another option: 

```suggestion
To create annotations, populate the widget's [annotations](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/annotations/) array. Each object in the array configures an individual annotation. To specify options for all annotations, use the [commonAnnotationSettings](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/commonAnnotationSettings/) object. Individual settings take precedence over common settings.
```